README.md

```diff
@@ -1 +1,106 @@
-# dosbox-staging
\ No newline at end of file
+# dosbox-staging
+[![Luxtorpeda project Discord](https://img.shields.io/discord/514567252864008206.svg?label=discord)](https://discord.gg/8mFhUPX)
+[![](https://github.com/dreamer/dosbox-staging/workflows/Linux%20builds/badge.svg)](https://github.com/dreamer/dosbox-staging/actions?query=workflow%3A%22Linux+builds%22)
+[![](https://github.com/dreamer/dosbox-staging/workflows/Windows%20builds/badge.svg)](https://github.com/dreamer/dosbox-staging/actions?query=workflow%3A%22Windows+builds%22)
+[![](https://github.com/dreamer/dosbox-staging/workflows/macOS%20builds/badge.svg)](https://github.com/dreamer/dosbox-staging/actions?query=workflow%3A%22macOS+builds%22)
+
+This repository is an attempt to revitalize the [DOSBox](https://www.dosbox.com/)
+project using modern tools and development practices. We're working on
+cleaning up, reviewing, and including community patches, developing new features,
+fixing existing issues, and sending revised patches upstream. Read more at 
+[Vogons thread](https://www.vogons.org/viewtopic.php?p=790065#p790065).
```

Comment:
"This repository attempts to modernize the DOSBox project by using current development practices and tools, fixing issues, adding features that better support today's systems, and sending patches upstream. Read more at Vogons thread."
(super minor-- switched to active-voice, dropped "reviewing" because it's part of current dev practices, combined "including community patches, developing new features" into the more generalized "adding features that better support today's systems" - because that can include both, plus more.. like adding new libraries, UI /graphical interfaces, and so on).